.gitignore

```diff
@@ -0,0 +1,4 @@
+venv
+idea
+sqlite3
+pyc
```

Comment:
Add empty line

Reply:
@vasylhnatiuk without "pyc" in .gitignore will be failure for checking test

Reply:
how do it in flake8, it working, no?